docker/runtime/julia/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian
+
+RUN apt-get update \
+ && apt-get install -y python3 python3-dev python3-pip python3-setuptools wget git bzip2 tar curl
```

Comment:
if this runtime is about performance I would avoid to use a Python wrapper (which will probably perform worse). I see that it is possible to use a web server with julia (https://github.com/JuliaWeb/HttpServer.jl) so I would try to do that before.

Reply:
but no julia prometheus_client is at hand.using a python wapper is a simple way.